modules/alloy/src/main/ts/ephox/alloy/ui/schema/TypeaheadSchema.ts

```diff
@@ -82,6 +83,10 @@ const parts: () => PartType.PartTypeAdt[] = Fun.constant([
               if (detail.model.populateFromBrowse) {
                 setValueFromItem(detail.model, input, item);
               }
+
+              // Since the focus is still on the typeahead comp, aria-activedescendant must be set to
+              // make screen readers to announce the menu item being highligted
+              Attribute.getOpt(item.element, 'id').each((id) => Attribute.set(input.element, 'aria-activedescendant', id));
```

Comment:
Here is where you set the attribute, within `onHighlightItem`, so removing it within `onDehighlightItem` makes more sense.